selfdrive/controls/lib/longitudinal_planner.py

```diff
@@ -1,31 +1,29 @@
 #!/usr/bin/env python3
 import math
 import numpy as np
-from common.params import Params
 from common.numpy_fast import interp
 
 import cereal.messaging as messaging
-from common.realtime import sec_since_boot
-from selfdrive.swaglog import cloudlog
+from common.realtime import DT_MDL
+from selfdrive.modeld.constants import T_IDXS
 from selfdrive.config import Conversions as CV
-from selfdrive.controls.lib.speed_smoother import speed_smoother
 from selfdrive.controls.lib.longcontrol import LongCtrlState
-from selfdrive.controls.lib.fcw import FCWChecker
+from selfdrive.controls.lib.lead_mpc import LeadMpc
 from selfdrive.controls.lib.long_mpc import LongitudinalMpc
-from selfdrive.controls.lib.drive_helpers import V_CRUISE_MAX
+from selfdrive.controls.lib.drive_helpers import V_CRUISE_MAX, CONTROL_N
 
 LON_MPC_STEP = 0.2  # first step is 0.2s
 AWARENESS_DECEL = -0.2     # car smoothly decel at .2m/s^2 when user is distracted
 
 # lookup tables VS speed to determine min and max accels in cruise
 # make sure these accelerations are smaller than mpc limits
-_A_CRUISE_MIN_V = [-1.0, -.8, -.67, -.5, -.30]
+_A_CRUISE_MIN_V = [-1.2, -1.2, -1.2, -1.2, -1.2]
```

Comment:
Just remove BPs if ther are all the same

Reply:
Tune isn't final yet.